zilliqa/src/exec.rs

```diff
@@ -454,13 +454,14 @@ impl State {
         &mut self,
         creation_bytecode: Vec<u8>,
         override_address: Option<Address>,
+        amount: Option<u128>,
```

Comment:
nit: `amount: None` doesn't make sense. Callers should just pass `0` if they don't want to send anything.

Reply:
Small fix, I'll do that now.